fv3core/fv3core/_config.py

```diff
@@ -64,6 +64,50 @@ class NamelistDefaults:
     nf_omega = 1
     fv_sg_adj = -1
     n_sponge = 1
+    # all below added for physics
```

Comment:
nit: it's going to be very easy for someone to add a non-physics value below, so you might want to consider removing the comment.

```suggestion
    # all below are for physics
```

Reply:
k thanks, I'll remove the comment